src/components/Footer/index.css

```diff
@@ -0,0 +1,206 @@
+@import "../../styles/all.css";
+
+$social-icon-size: 28px;
+$social-icon-size-mobile: 16px;
```

Comment:
Even if those are icon sizes, shouldn't we use spacings?

Reply:
Spacings are space between thing, image sizes are other (sizes can be related in some cases) aspect

Reply:
But i can use spacings in fact, in this case there is a correlation, but on the team photos for example, not so much

Reply:
👍